recipes/platform.interfaces/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.interfaces"
+    license = "The Unlicense"
+    homepage = "https://github.com/linksplatform/Interfaces"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Contains common interfaces that did not fit in any major category."
+    topics = ("platform", "concepts", "header-only")
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return self.name
+
+    @property
+    def _subfolder_sources(self):
+        return os.path.join(self._source_subfolder, "cpp", "Platform.Interfaces")
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "10",
+            "Visual Studio": "16",
+            "clang": "11",
+            "apple-clang": "11"
+        }
+
+    def validate(self):
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler))
+        if tools.Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"platform.interfaces/{self.version} "
+                                            f"requires C++20 with {self.settings.compiler}, "
+                                            f"which is not supported "
+                                            f"by {self.settings.compiler} {self.settings.compiler.version} ")
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 20)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+        url: str = self.conan_data["sources"][self.version]["url"]
+        version_pos: int = url.find(f"_{self.version}")
+        csharp_version: str = url[(version_pos - 5):version_pos]

```

Comment:
As long as we are using Conan v1.x to build the recipes we want to keep them compatible with Python 2. So, please, do not use type annotations:

```suggestion
        url = self.conan_data["sources"][self.version]["url"]
        version_pos = url.find(f"_{self.version}")
        csharp_version = url[(version_pos - 5):version_pos]

```